src/app.js

```diff
@@ -1,5 +1,202 @@
 const express = require('express');
+const { sayHello, uppercase, lowercase, firstCharacters } = require('./lib/strings');
+const { add, subtract, multiply, divide, remainder } = require('./lib/numbers');
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement,
+} = require('./lib/arrays');
 
 const app = express();
 
+app.use(express.json());
+
```

Comment:
For long files like this, you could break up the large block of code with heading comments like:
```

// String methods

```